### PR TITLE
Feature/sc 2672/deploy nftx v3 frontend to base

### PR DESCRIPTION
**Pull request by jackmellis**

### CODE REVIEW

File: packages/core/src/assets/fetchUserHoldings/fetchErc721s.ts [hunk 1]
```diff
@@ -69,37 +64,13 @@ export const makeFetchErc721s =
...
-          lastId,
-          network,
-          userAddress,
-          querySubgraph,
-        });
-        break;
-      default:
-        break;
-    }
+    const [holdings, nextId] = await fetchErc721s({
```
Comment (0xNumlock) [inline packages/core/src/assets/fetchUserHoldings/fetchErc721s.ts:68]: Nice, much cleaner.

File: packages/core/src/vaults/fetchVaultActivity.ts [hunk 2]
```diff
@@ -273,7 +273,7 @@ export const makeFetchVaultActivity = ({
     const activity: VaultActivity[] = [];
     const feeDistributorAddresses = [
       getChainConstant(NFTX_FEE_DISTRIBUTOR, network),
-      ...getChainConstant(NFTX_FEE_DISTRIBUTOR_LEGACY, network),
+      ...getChainConstant(NFTX_FEE_DISTRIBUTOR_LEGACY, network, []),
```
Comment (0xNumlock) [inline packages/core/src/vaults/fetchVaultActivity.ts:276]: Can't we just make the last param optional instead of passing in empty values?

**Review by 0xNumlock — APPROVED** [review]
Awesome, small comment but no blockers - brilliant work as always!

File: packages/core/src/vaults/fetchVaultActivity.ts (hunk 2 above)
Comment (jackmellis) [reply packages/core/src/vaults/fetchVaultActivity.ts:276]: `getChainConstant` is a generic fn that we use all over and the return value is also generic, so if you want a fallback value, you have to pass it in (or it returns undefined).

**Pull request merged**